CoreFoundation/Collections.subproj/CFArray.c

```diff
@@ -628,7 +628,7 @@ void CFArrayAppendValue(CFMutableArrayRef array, const void *value) {
 }
 
 void CFArraySetValueAtIndex(CFMutableArrayRef array, CFIndex idx, const void *value) {
-    CF_SWIFT_FUNCDISPATCHV(CFArrayGetTypeID(), void, (CFSwiftRef)array, NSMutableArray.setObject, idx, value);
+    CF_SWIFT_FUNCDISPATCHV(CFArrayGetTypeID(), void, (CFSwiftRef)array, NSMutableArray.setObject, value, idx);
```

Comment:
We should really name all the argument in order in the bridge struct field to prevent this from occurring again. Is there any way I can convince you to add an edit so this reads NSMutableArray.setObjectAtIndex?

✅ whether this is done or not.